### ISSUE TITLE
FIXME: Mismatch between warning and errors

### ISSUE THREAD

**Issue by hk21702** [RimSort/RimSort#502]
**Describe the bug:**
When a mod has an error such as a missing dependency detected, the icon next to it is the warning icon. The error count at the bottom is correctly bumped and the error is correctly added into the tooltip. The text of the mod is also not red/highlighted.

**Screenshots:**
![image](https://github.com/user-attachments/assets/87db77c8-2184-4207-8496-67ac2f523cfe)
![image](https://github.com/user-attachments/assets/effca558-2a24-4fbc-a875-e5ca2c016773)
![image](https://github.com/user-attachments/assets/7cb362aa-e0f7-4cb7-bfa4-32eb0e74339d)

**Please complete the following information:**

- Operating System: Windows
- RimSort version: 1.0.9-edge92

**Additional context:**

**Comment by Patryk-Malinowski** [comment]
Quick question, should a mod with a warning be red?

Or should warnings only have the warning icon and errors are red with the error icon?

**Comment by hk21702** [comment]
Current behavior is that warnings are also red. For example, version mismatch counts as a warning and results in red text, a warning icon, and a warning added to the bottom. I'd say just keep current behavior on that front.

**Comment by Patryk-Malinowski** [comment]
Ok will do, cheers!

**Comment by Patryk-Malinowski** [comment]
When a mod has a warning because of load order, should it be colored red? The current behavior is that its not colored.
![image](https://github.com/user-attachments/assets/94a34471-530c-4068-adde-c32b3ea46f17)

**Comment by hk21702** [comment]
For consistency probably? Given that it counts as a warning, I'd presume it should be the same as other warnings.
